client_test.go

```diff
@@ -53,3 +53,9 @@ func TestSubmitPDF(t *testing.T) {
 	assert.NoError(t, err)
 	assert.NotNil(t, txRes)
 }
+
+func TestBalance(t *testing.T) {
+	//t.Skip()
```

Comment:
```suggestion
```

Reply:
Just a note: #4 needs to be merged to enable tests.